app/data_model/database.py

```diff
@@ -21,21 +20,12 @@ class QuestionnaireState(base):
     user_id = Column('userid', String, primary_key=True)
     state = Column('questionnaire_data', String)
 
-    def __init__(self, user_id, data):
+    def __init__(self, user_id, state):
         self.user_id = user_id
-        self.state = json.dumps(data)
-
-    def set_data(self, data):
-        logger.debug("setting data for questionnaire state")
-        self.state = json.dumps(data)
-
-    def get_data(self):
-        data = json.loads(self.state)
-        logger.debug("loading questionnaire state")
-        return data
+        self.state = state
 
     def __repr__(self):
-        return "<QuestionnaireState('%s','%s')>" % (self.user_id, self.state)
+        return "<QuestionnaireState('%s','%s')>" % (self.user_id, self._state)
```

Comment:
Do we have unit tests that check this function works when calling `repr()` ?

Reply:
no, https://codecov.io/gh/ONSdigital/eq-survey-runner/src/master/app/data_model/database.py

Reply:
Should add some then...

Reply:
Looking at it, this code will actually fail as the variable is named incorrectly _state